src/odemis/gui/cont/export.py

```diff
@@ -115,6 +115,7 @@ def on_streams_change(self, streams):
         enabled = (len(streams) > 0)
         self._main_frame.menu_item_export_as.Enable(enabled)
         self._tab_panel.btn_export.Enable(enabled)
+        self._tab_panel.btn_export.SetLabel(f"Export Image ({len(streams)})")
```

Comment:
If there are no streams, it shows a disabled button with label "Export Image (0)". I think it sounds odd. Probably the label should just be "Export Image" in such cases.